### DISCUSSION TITLE
Add Timezone support

### DISCUSSION THREAD

**Discussion by jhoff** [spatie/laravel-health#252] [Ideas]
Had a teammate see the `timezone()` method and expected it to work like the Kernel scheduler, but instead our log files just started filling up with `Creation of dynamic property CheckClass::$timezone is deprecated` warning on every single request and the checks still ran in UTC, despite the code suggesting otherwise.

`Spatie\Health\Checks\Check` uses `Illuminate\Console\Scheduling\ManagesFrequencies` which offers the ability to set `$timezone`. `Spatie\Health\Checks\Check::shouldRun()` could easily support timezones in the same way that `Illuminate\Console\Scheduling\Event::expressionPasses()` does now.

**Comment by freekmurze** [comment]
I'd accept a PR (with tests) for this 👍

**Comment by jhoff** [comment]
Thanks @mateusjatenee https://github.com/spatie/laravel-health/pull/264

**Comment by jhoff** [comment]
This was merged! Closing
